cli/tests/integration/repl_tests.rs

```diff
@@ -801,7 +801,7 @@ fn repl_error() {
   });
 }
 
-#[test]
+#[flaky_test::flaky_test]
```

Comment:
I'm not sure it's worth the effort to fix these, but basically I think the output gets cleared by rustyline and cuts off some of the message that displays. For example, sometimes this test displays:

```
Uncaught (in promise) Error: foo
> 
```

Instead of:

```
Uncaught (in promise) Error: foo
    at <anonymous>
> 
```

I think that happens because the reporting on that "Uncaught (in promise)" happens a bit later?